app/src/androidTest/java/com/github/se/signify/ui/screens/home/HomeScreenTest.kt

```diff
@@ -171,16 +190,34 @@ class HomeScreenTest {
 
     letters.forEach { letter ->
       // Assert that the letter and corresponding icon are displayed
-      composeTestRule.onNodeWithTag("LetterText_$letter").assertIsDisplayed()
-      composeTestRule.onNodeWithTag("LetterIcon_$letter").assertIsDisplayed()
+      composeTestRule.onNodeWithTag("LetterBox_$letter").assertIsDisplayed()
 
       // Click on the back arrow to go to the previous letter
       composeTestRule.onNodeWithTag("LetterDictionaryBack").performClick()
     }
 
     // After looping through all letters, it should go back to 'Z'
-    composeTestRule.onNodeWithTag("LetterText_Z").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("LetterIcon_Z").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("LetterBox_Z").assertIsDisplayed()
+  }
+
+  @Test
+  fun dictionaryIsDisplayed() {
+    composeTestRule.setContent { HomeScreen(navigationActions) }
```

Comment:
The HomeScreen initialization repeats in multiple tests. I think it will be better to move it to the @Before function, centralizing it for reusability and reducing duplication.

Reply:
Thomas wrote this class, it is true that it's duplicated but there are different tests that isolate their composables.
`i.e compose.TestRule.setContent{???}`